upgrade/steps.go

```diff
@@ -324,38 +322,32 @@ func InformGitHub(
 	} else if ctx.UpgradeSdkVersion {
 		prTitle = "Upgrade Pulumi SDK dependency"
 	} else {
-		panic("Unknown action")
+		return fmt.Errorf("Unknown action")
 	}
 
-	createPR := step.Cmd("gh", "pr", "create",
+	stepv2.Cmd(ctx, "gh", "pr", "create",
 		"--assignee", GetContext(ctx).PrAssign,
 		"--base", repo.defaultBranch,
 		"--head", repo.workingBranch,
 		"--reviewer", GetContext(ctx).PrReviewers,
 		"--title", prTitle,
-		"--body", prBody(ctx, repo, target, goMod, targetBridgeVersion, tfSDKUpgrade, os.Args),
-	).In(&repo.root)
-	return step.Combined("GitHub",
-		pushBranch,
-		createPR,
-		step.Computed(func() step.Step {
-			// If we are only upgrading the bridge, we wont have a list of
-			// issues.
-			if !GetContext(ctx).UpgradeProviderVersion {
-				return nil
-			}
+		"--body", prBody(ctx, repo, target, goMod, targetBridgeVersion, tfSDKUpgrade, osArgs))
 
-			// This PR will close issues, so we assign the issues same assignee as
-			// the PR itself.
-			issues := make([]step.Step, len(target.GHIssues))
-			for i, t := range target.GHIssues {
-				issues[i] = step.Cmd(
-					"gh", "issue", "edit", fmt.Sprintf("%d", t.Number),
-					"--add-assignee", GetContext(ctx).PrAssign).In(&repo.root)
-			}
-			return step.Combined("Assign Issues", issues...)
-		}),
-	)
+	// If we are only upgrading the bridge, we wont have a list of issues.
+	if !GetContext(ctx).UpgradeProviderVersion {
+		return nil
+	}
+
+	stepv2.Call00(ctx, "Assign Issues", func(ctx context.Context) {
```

Comment:
It kind of seems like `Call00` should have a special name; iiuc, this is just providing a grouping with a label for the steps inside right?